decidim-conferences/app/commands/decidim/conferences/admin/create_conference_speaker.rb

```diff
@@ -25,18 +25,54 @@ def initialize(form, current_user, conference)
         def call
           return broadcast(:invalid) if form.invalid?
 
-          transaction do
-            create_conference_speaker!
-            link_meetings(@conference_speaker)
-          end
+          # We are using going to assign the attributes only to handle the validation of the avatar before accessing
+          # create_conference_speaker! which uses create! with bang, and this will render an ActiveRecord::RecordInvalid error
+          # After we assign and check if the object is valid, then we will not save the model to let it be handled the old way
+          # If there is an error we add the error to the form
+          # We are using this method to assign the conference as if we are trying to assign all at once, there will be thrown a
+          # Delegation error
+
+          if conference_speaker_with_attributes.valid?
+            # Do not persist this. Please see the comment above!
+            # conference_speaker_with_attributes.save
```

Comment:
Remove this?
```suggestion
```

Reply:
This was left there on purpose, to make sure nobody will come up with the idea to add that save

Reply:
I don't think it's needed. The comment above already explains why we should persist the object.